tests/pytorch/test_fused_attn.py

```diff
@@ -538,10 +696,6 @@ def _run_dpa_fp8_ref(dtype, bs, config, backend):
 
     os.environ["NVTE_FLASH_ATTN"] = "0"
     os.environ["NVTE_FUSED_ATTN"] = "0"
-    if backend == "FlashAttention":
-        os.environ["NVTE_FLASH_ATTN"] = "1"
-    if backend == "FusedAttention":
-        os.environ["NVTE_FUSED_ATTN"] = "1"
```

Comment:
I think these `if backend ==` statements should be kept here, because it's in the `_run_dpa_fp8_ref` function that we call DPA, and the `_run_dpa_fp8` calls something custom (`DPA_FP8()` defined in this test).